hamocc/mo_inventory_bgc.F90

```diff
@@ -901,8 +901,7 @@ subroutine write_netcdf(iogrp)
         endif
         write(tstamp,'(i4.4,a1,i2.2,a1,i2.2)')                                    &
              &    date%year,sep2,date%month,sep2,date%day
-        fname_inv(iogrp) = prefix//sep1//trim(glb_fnametag(iogrp))//sep1//        &
-             &    'i'//sep1//trim(tstamp)//'.nc'
+        fname_inv(iogrp) = prefix//sep1//'hbgci'//sep1//trim(tstamp)//'.nc'
```

Comment:
Should likely read:
```
call blom_time(ymd,tod)
write(tstamp,'(i4.4,a1,i2.2,a1,i2.2,a1,i5.5)') date%year,sep2,date%month,sep2,date%day,sep2,tod
fname_inv(iogrp) = prefix//sep1//'bgci'//sep1//trim(tstamp)//'.nc'
```
(requires `integer :: ymd,tod` and `use mod_time, only: blom_time`